### ISSUE TITLE
Persist Extended Settings Across Registered User Sessions

### ISSUE THREAD

**Issue by demory** [conveyal/modeify#314]
Earlier today when reviewing the app w/ Neoniche (who are preparing to start a round of one-on-one field outreach w/ CFAZ) it was pointed out that the new extended walk/bike/drive preferences are not persisting across sessions for registered / logged-in users. The ability to save settings is being touted as a benefit of registering so it would be good to add this.

Regarding implementation -- @trevorgerhardt is this what the "opts" field in the commuter model is intended for?

**Comment by trevorgerhardt** [comment]
Yup. It was being persisted at one point. The switch to the new settings or changes with Stormpath may have caused it to change.

**Comment by demory** [comment]
OK thanks. I'll take a look at this.

**Comment by trevorgerhardt** [comment]
@demory you completed this correct?
